### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by alex-golts** [GIST-CSBL/HoTS#4]
Hello, 
Thank you for your great work on the HoTS paper and model and for open sourcing it for the community!
I noticed that you have removed the trained model from the repo. Is that intended?
If it is, please consider updating the "prediction with trained model" section in the readme to reflect that.

Thank you,
Alex

**Comment by dlsrnsi** [comment]
Hello, Alex

Thank you for your interest in my research.
Actually, now I'm converting my model to tensorflow version 2.4.1, and training the model again. 

The trained model will be uploaded soon (guess this weekend), then you can use it freely.

Thank you for Biopython too, I will add it to my environment.

Also, I'm gonna make the branch of tf1 from previous commit.

Again, thank you for your interest in my model.

Sincerely,

Ingoo Lee

**Comment by alex-golts** [comment]
This sounds great, really looking forward to trying it out!

**Comment by dlsrnsi** [comment]
Dear @alex-golts 

Sorry for being late,

I updated the code and jupyter notebook

Also, I uploaded the models, and made the tf1 branch.

For prediction, I recommend using tf1-version.

tf2 version used different normalization because of technical use.

[https://www.kaggle.com/questions-and-answers/188767](https://www.kaggle.com/questions-and-answers/188767)

Thank you for your interest in my model 

Sincerely, 

Ingoo Lee

**Comment by alex-golts** [comment]
thanks a lot for the heads up! will definitely want to check it out...

FYI, the link in the main README that says "the trained model" links to [this URL](https://github.com/GIST-CSBL/HoTS/blob/master/Model/HoTS_config.json) which is still broken. but I am able to see the model in the code structure!

**Comment by dlsrnsi** [comment]
Sorry for replying late, I've been busy with a thesis defense and job applications.

I changed the URL so you can access through it
